lib/tests/CMakeLists.txt

```diff
@@ -22,9 +22,11 @@ set(test_BIN cpp-pcp-client-unittests)
 find_package(Boost 1.54 REQUIRED
   COMPONENTS filesystem system date_time thread log regex random)
 
+# TODO(ale): remove Thread dependency after LTH-81 changes
 include_directories(
     ${LEATHERMAN_CATCH_INCLUDE}
     ${Boost_INCLUDE_DIRS}
+    ${CMAKE_THREAD_LIBS_INIT}
```

Comment:
@parisiale I think we put this in the wrong place. I'll put up a new PR once I verify on AIX.